packages/synapse-react-client/src/components/widgets/query-filter/CombinedRangeFacetFilter.tsx

```diff
@@ -45,24 +48,50 @@ export const CombinedRangeFacetFilter: React.FunctionComponent<
   onChange,
   collapsed = false,
 }: CombinedRangeFacetFilterProps) => {
+  const { getCurrentQueryRequest } = useQueryContext()
+
   const [isCollapsed, setIsCollapsed] = useState<boolean>(collapsed)
   const {
-    columnMin: col1Min,
+    columnName: col1Name,
     // columnMax: col1Max  // not used
     selectedMin: col1SelectedMin,
     selectedMax: col1SelectedMax,
   } = facetResults[0]
   const {
+    columnName: col2Name,
     //columnMin: col2Min,  // not used
-    columnMax: col2Max,
     selectedMin: col2SelectedMin,
     selectedMax: col2SelectedMax,
   } = facetResults[1]
 
+  // Also query for the facet columns full range (without any selected facets), to set the range selector range
+  const queryBundleRequest = useMemo(() => {
+    const requestCopy = getCurrentQueryRequest()
+    requestCopy.partMask = SynapseConstants.BUNDLE_MASK_QUERY_FACETS
+    const { selectedFacets } = requestCopy.query
+    requestCopy.query.selectedFacets = selectedFacets
+      ? selectedFacets.filter(
+          facetRequest =>
+            facetRequest.columnName !== col1Name &&
+            facetRequest.columnName !== col2Name,
+        )
+      : []
+    return requestCopy
+  }, [col1Name, col2Name, getCurrentQueryRequest])
+
+  const { data: queryResultFacetResponse, isLoading: isLoadingFacetStats } =
+    useGetQueryResultBundleWithAsyncStatus(queryBundleRequest)
+
+  const fullFacetStats = queryResultFacetResponse?.responseBody?.facets
+
+  const col1Facet = fullFacetStats?.find(facet => facet.columnName === col1Name)
+  const col2Facet = fullFacetStats?.find(facet => facet.columnName === col2Name)
+  const col1MinMin = (col1Facet as FacetColumnResultRange)?.columnMin
+  const col2MaxMax = (col2Facet as FacetColumnResultRange)?.columnMax
```

Comment:
It's just the column minimum/maximum, but ignores any facet selection from either column1 or column2.  Instead of col1MinMin, what about col1GlobalMin (to suggest this idea)?